lib/prmd/version.rb

```diff
@@ -2,7 +2,7 @@
 module Prmd
   # Well, duh, its a Version module, what did you expect?
   module Version
-    MAJOR, MINOR, TEENY, PATCH = 0, 7, 4, nil
+    MAJOR, MINOR, TEENY, PATCH = 0, 7, 4, 1
```

Comment:
Could you omit the version change in your patch? I'll do that independently when I release.